.github/workflows/bootstrap.yml

```diff
@@ -0,0 +1,76 @@
+# https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions
+
+name: Bootstrap
+on:
+  pull_request:
+    branches:
+      - stable
+  push:
+    branches:
+      - master
+      - stable
```

Comment:
I thought you wanted it the other way around? All PRs, but only pushes to stable, to skip it for master-merges?

Reply:
My thinking was, I'd rather not burden _every_ PR by requiring them to keep backwards compatibility with every compiler.  Running on all pushes to master at least allows anyone looking to keep tabs on the build state during development.

Running all pipelines on PRs to stable helps to ensure all releases are buildable by any given old release is the desirable state be in.

As these pipelines are cheap compared to the regular DMD full testsuite iobs, I can't imagine it being a problem for master merges.  As of writing we're unlikely to create a massive backlog on the free resources given GitHub's generous quotas (for now).

Reply:
Yeah I don't have a strong opinion on this. As long as someone does check the master state from time to time, we should be fine. I was just mildly worried about merging master into stable potentially leading to a red CI because of bootstrap checks, something I know from experience isn't a nice experience for the release manager. :)